src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -24,30 +24,27 @@ var ownerRepoIssueRegex = regexp.MustCompile("([A-z0-9-_]+)/([A-z0-9-_]+)#([0-9]
 
 type githubService struct {
 	id                 string
+	serviceUserID      string
 	webhookEndpointURL string
-	BotUserID          string
 	ClientUserID       string
 	RealmID            string
 	SecretToken        string
+	HandleCommands     bool
+	HandleExpansions   bool
```

Comment:
Not planning to do that right now given that commands/expansions are already efficient (simple SELECT based on the user_id of the client doing the `/sync`). Maybe in the future if we need to.